src/wp-includes/class-wp-image-editor.php

```diff
@@ -428,15 +433,18 @@ protected function get_output_format( $filename = null, $mime_type = null ) {
 	 * Builds an output filename based on current file, and adding proper suffix
 	 *
 	 * @since 3.5.0
+	 * @since 6.1.0 Skips adding a suffix when set to an empty string.
 	 *
-	 * @param string $suffix
+	 * @param string $suffix Optional. Suffix to add to the filename. Passing null
+	 *                       will result in a 'widthxheight' suffix. Passing
+	 *                       an empty string will result in no suffix.
 	 * @param string $dest_path
 	 * @param string $extension
 	 * @return string filename
```

Comment:
@adamsilverstein Nitpick, but since we've now added documentation for `$suffix`, it may be good to document the other parameters and the return value as well.

Reply:
Documented in [`feaea78` (#2393)](https://github.com/WordPress/wordpress-develop/pull/2393/commits/feaea78a9cedcb309d1295602545bee6e51042bb)